crates/re_examples/src/lib.rs

```diff
@@ -0,0 +1,171 @@
+//! Example collection and parsing.
+
+use std::fmt::Display;
+use std::path::Path;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+pub struct Example {
+    pub name: String,
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<String>,
+    pub thumbnail_url: String,
+    pub thumbnail_dimensions: [u64; 2],
+    pub script_path: PathBuf,
+    pub script_args: Vec<String>,
+    pub kind: ExampleKind,
```

Comment:
this could be `pub channel: Option<Channel>` to reuse the enum and name.
Alternatively, add `None` as an alternative in `Channel` and use `pub channel: Channel` here